h/assets.yaml

```diff
@@ -155,26 +155,8 @@ app_css:
       contents:
         - h:static/styles/app.scss
       depends:
```

Comment:
We don't want `inject.scss` in this bundle.

We should consider starting to group our bundles on the filesystem, but right now the parallel construction we have suggests we should just move `inject.scss` into a `styles/annotator` folder.